Wellthier/View Controllers/ComposeViewController.h

```diff
@@ -0,0 +1,22 @@
+//
+//  ComposeViewController.h
+//  Wellthier
+//
+//  Created by Craig Lee on 7/19/22.
+//
+
+#import "ViewController.h"
```

Comment:
Not sure if we really Need to Import viewController.

Reply:
we do as composeviewcontroller is a subclass of viewcontroller

Reply:
causes error if not imported